piptools/repositories/pypi.py

```diff
@@ -22,7 +22,7 @@
 
 
 class PyPIRepository(BaseRepository):
-    DEFAULT_INDEX_URL = 'https://pypi.python.org/simple'
+    DEFAULT_INDEX_URL = 'https://pypi.org/simple/'
```

Comment:
The error is caused by adding a trailling `/` here, in conjuction with this: https://github.com/jazzband/pip-tools/blob/fe8c30e14201abf46f5639a79b65f56398b6ff9f/piptools/writer.py#L58

The result is that the index-url is added to the output as if it wasn't the default one. Removing the trailling `/` here will restore the behavior.

Reply:
Thanks for looking at the failure. I've made this change. However, it looks like tests are still failing.

Reply:
I guess the configured index on the BMs is still the old one, or it's in the vendored pip code.